runtime/crab/src/migration.rs

```diff
@@ -41,7 +41,16 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 }
 
 fn migrate() -> frame_support::weights::Weight {
-	frame_support::weights::Weight::zero()
+	// substrate
+	use frame_support::traits::StorageVersion;
```

Comment:
These changes are also needed for the darwinia runtime?

Reply:
No

Reply:
```sh
2023-08-30 13:39:11.238  INFO main runtime::frame-support: ⚠️ XcmpQueue declares internal migrations (which *might* execute). On-chain `StorageVersion(3)` vs current storage version `StorageVersion(2)`
2023-08-30 13:39:11.238  INFO main runtime::frame-support: ⚠️ DmpQueue declares internal migrations (which *might* execute). On-chain `StorageVersion(2)` vs current storage version `StorageVersion(1)`
```
I analysed the darwinia try-runtime result and found that these two pallet also break the try-runtime process. If we do nothing at all, the darwinia runtime try-runtime still panic if I understand correctly.

Reply:
Please check the audit report.